.github/workflows/unit-tests.yml

```diff
@@ -17,26 +13,257 @@ permissions:
 
 jobs:
 
-  tests:
-    name: Unit Tests
+# linux:
+#   name: Linux
```

Comment:
A simpler way to disable jobs without generating such a huge diff that makes it harder to see what are the actual changes is to add `if: false` in the job definition (i.e. making it a conditional job, with a condition that is always false)